alacritty_terminal/src/term/mod.rs

```diff
@@ -2429,6 +2733,277 @@ mod tests {
         assert_eq!(term.grid.cursor.point, Point::new(Line(4), Column(0)));
     }
 
+    #[test]
+    fn damage_public_usage() {
+        let size = SizeInfo::new(10.0, 10.0, 1.0, 1.0, 0.0, 0.0, false);
+        let mut term = Term::new(&Config::default(), size, ());
+        // Reset terminal for partial damage tests since it's initialized as fully damaged.
+        term.reset_damage();
+
+        // Test that we damage input form [`Term::input`].
```

Comment:
```suggestion
        // Test that we damage input form [`Term::input`].
        
```

This comment isn't directly related to the code in annotates, but instead applies to everything that follows. In that case I find it helpful to add a newline between the comment and the actual content.

This also applies to the rest of the function and comments in it.